wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md

```diff
@@ -40,23 +40,23 @@ Images
 
 There are 2 major image formats which make up most of the images you find online: jpg and png. Png uses lossless compression which makes it ideal for small and or simple images. Complex images are better suited for jpg. The majority of image editing programs however do not naturally use `.png` and `.jpg` compression to the most of their ability however, leaving you with 4 KB pngs of 1 color, which could be compressed up to 400 times smaller.
 
-[Gimp](http://www.gimp.org/) actually does a good job with `.jpg` files.
+[GIMP](http://www.gimp.org/) actually does a good job with `.jpg` files.
 
-![](CBF_02.png "CBF_02.png")
+![Compressing Images](CBF_02.png "Compressing Images")
 
 There are four important things to check: Optimize, Progressive, subsampling, and quality. Optimize is just a way to compress the `.jpg` without losing quality and should always be checked. Progressive is a method to make jpgs "fade in" when downloaded from the internet. It has a side effect of making the image smaller so long as the image is over 20 KB. Subsampling is a way to improve quality or compression. No subsampling (1x1) will make text and lines much clearer at the cost of space. 2x2 subsampling does very well for photographs which do not have distinct lines (like text). Quality is the easiest way to keep the image looking good. I use 95, since the gains after 95 aren't really worthwhile.
 
-Pngs are actually much simpler to compress, but GIMP does not do it very well. [Optipng](http://optipng.sourceforge.net/) and [Pngout](http://advsys.net/ken/utils.htm) are two of the best tools to compress `.png` files. Optipng can find the optimum settings for the `.png`, and pngout re-compresses the image using `.kzip`. Often times just using pngout will reduce the image size by 20%.
+Pngs are actually much simpler to compress, but GIMP does not do it very well. [Optipng](http://optipng.sourceforge.net/) and [PNGout](http://advsys.net/ken/utils.htm) are two of the best tools to compress `.png` files. Optipng can find the optimum settings for the `.png`, and pngout re-compresses the image using `.kzip`. Often times just using pngout will reduce the image size by 20%.
```

Comment:
Change `Pngs` to `PNGs`
Change `PNGout` to PNGOUT` - this is the way how the program is spelt

---

I assume Capuccino meant the first instance of `Pngs` that you should change that to `PNGs`. Also the program (`PNGout`) is all caps.